### DISCUSSION TITLE
Preload webfonts?

### DISCUSSION THREAD

**Discussion by it-can** [BabDev/laravel-server-push-manager#2] [Q&A] [answered]
Is it possible to preload webfonts?

**Comment by mbabker** [comment, marked as answer]
Yes.  `PushManager::preload(asset('fonts/BPscript.woff2'), ['as' => 'font', 'type' => 'font/woff2']);`

I'm doing it in https://github.com/BabDev/babdev.com/blob/production/app/Http/Middleware/PreloadAssets.php

**Reply by it-can** [reply]
aaaah cool :-)
